yarn-project/stdlib/src/noir/index.ts

```diff
@@ -75,7 +75,7 @@ export interface NoirCompiledContract {
 }
 
 /**
- * The compilation result of an Aztec.nr contract.
+ * The compilation result of a protocol (non-contract) circuit.
```

Comment:
This was incorrect as it doesn't seem to ever be used with contracts. There is a lot of tech debt around.